engine/table/src/main/java/io/deephaven/engine/table/impl/by/ChunkedOperatorAggregationHelper.java

```diff
@@ -1619,12 +1654,13 @@ private static void initialBucketedKeyAddition(QueryTable withView,
                 for (int ii = 0; ii < ac.size(); ++ii) {
                     final int inputSlot = ac.inputSlot(ii);
                     if (ii == inputSlot) {
-                        if (!findRuns) {
+                        if (!permute) {
                             valueChunks[inputSlot] = getChunk(ac.inputColumns[ii], getContexts[ii], chunkOk, usePrev);
                         } else {
-                            getAndPermuteChunk(ac.inputColumns[ii], getContexts[ii], chunkOk, usePrev,
-                                    permuteKernels[ii],
-                                    chunkPosition, workingChunks[ii]);
+                            assert workingChunks != null;
```

Comment:
Is this assertion intended for commit? Should be it be an `Assert.assertion` if so?

Reply:
It makes IntelliJ happy that workingChunks[ii] isn't going to be an NPE to reduce a yellow warning.  Our own assertion doesn't do that, because IJ doesn't know about it.  I can take it out if you prefer.